### ISSUE TITLE
PR #61 broke reading from values file

### ISSUE THREAD

**Issue by MindTooth** [renovatebot/helm-charts#62]
I believe #61  to have caused my working `values.yaml` file to break.  I've attached it below.  Working with 23.31.2, the latest release before the PR got merged.

Error message:
```log
Error: YAML parse error on renovate/templates/secret.yaml: error converting YAML to JSON: yaml: line 15: could not find expected ':'
```

Tried various combinations of YAML/JSON, but the error still persists.  Did also look over the PR to see if I could catch the error, but was too much going on for me to notice anything.

Any advice?

---

```yaml
cronjob:
  schedule: "0 */6 * * *"
  concurrencyPolicy: Forbid
  jobRestartPolicy: Never

secrets:
  GITHUB_COM_TOKEN: "$GITHUB_API_TOKEN"
  LOG_LEVEL: "debug"
  RENOVATE_AUTODISCOVER: "true"
  RENOVATE_ENDPOINT: ### URL ###
  RENOVATE_PLATFORM: "gitlab"
  RENOVATE_TOKEN: "$GITLAB_RENOVATE_USER"

renovate:
  config: |
    {
      "onboardingConfig": {
        "extends": ["config:base",":label(renovate)"]
      },
      "requireConfig": true
    }

```

**Comment by davidholsgrove** [comment]
Hi @MindTooth - PR should resolve the issue with inline secret data. Note the example you gave will require a non-comment value for `RENOVATE_ENDPOINT` for the yaml to base64 encoding to work also.

**Comment by MindTooth** [comment]
The PR did break something.  It used to work before the PR and now it just fails no matter what the config looks like.

`### URL ###` line is already quoted in the config file.  Just got stripped in the example.

I had this before, and that worked too:
```yaml
secrets:
  {
    GITHUB_COM_TOKEN: "$GITHUB_API_TOKEN",
    LOG_LEVEL: "debug",
    RENOVATE_AUTODISCOVER: "true",
    RENOVATE_ENDPOINT: "### URL ###",
    RENOVATE_PLATFORM: "gitlab",
    RENOVATE_TOKEN: "$GITLAB_RENOVATE_USER",
  }
```

**Comment by JamieMagee** [comment]
@MindTooth Due to how we've chosen to version the helm chart (it aligns with the renovate version) the fix made in #63 won't be pushed out until there is an update in in the Docker image. Apologies about this.

@davidholsgrove Thanks for the quick turnaround on the fix here.

**Comment by MindTooth** [comment]
Woot!

Closing in anticipation of #63.
